vanity-pet.lic

```diff
@@ -1,94 +1,94 @@
-=begin

```

Comment:
⚡ I'm not sure about what changes are being made here? Either way, I don't think this file should be part of this PR, correct? This was recently introduced in https://github.com/rpherbig/dr-scripts/pull/4678

Reply:
This file is still showing as deleted in the PR